clj-refactor.el

```diff
@@ -761,14 +761,17 @@ A new record is created to define this constructor."
 ;; ------ file -----------
 
 (defun cljr--project-dir ()
-  (or (ignore-errors
-        (file-truename
-         (locate-dominating-file default-directory "project.clj")))
-      (ignore-errors
-        (file-truename
-         (locate-dominating-file default-directory "build.boot")))
-      (ignore-errors (file-truename
-                      (locate-dominating-file default-directory "pom.xml")))))
+  (let ((project-dir (or (ignore-errors
+			   (file-truename
+			    (locate-dominating-file default-directory "project.clj")))
+			 (ignore-errors
+			   (file-truename
+			    (locate-dominating-file default-directory "build.boot")))
+			 (ignore-errors (file-truename
+					 (locate-dominating-file default-directory "pom.xml"))))))
+    (if (null project-dir)
```

Comment:
perhaps rather `(or project-dir "")`?

or in fact you can get rid of the `let` you introduced and just stick an empty string at the end of the original `or` expression as default?